modules/custom/improved_theme_settings/improved_theme_settings.module

```diff
@@ -184,3 +173,16 @@ function improved_theme_settings_page_attachments(array &$page) {
     }
   }
 }
+
+/**
+ * Function that returns a valid integer or FALSE.
+ */
+function improved_theme_settings_get_setting($setting, $themename) {
+  // Fetch setting from the theme.
+  $themevar = Xss::filter(theme_get_setting($setting, $themename));
+  // In this case '' is something different than 0.
+  if ($themevar !== trim(intval($themevar))) {
```

Comment:
@jochemvn 
I think this change causes a regression from a previous version. E.g. `'5 '` would previously be counted as a valid value because it'd be parsed to `5` but `trim(intval('5 '))` evaluates to `'5'` which does not equal `'5 '`. 

If the setting ends up being an object due to a misconfigured subtheme this code will also produce a notice instead of disabling the feature without errors.

Reply:
No it doesn't. The field is a number field, you cannot add values like '5 ' there.

Reply:
There is no validation (like a config schema) for that anywhere else so currently it's allowed for a subtheme to have the value '5'. This is why a simple `is_null || !is_integer` did not suffice.